jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -1337,29 +1337,41 @@ ecma_string_raw_chars (const ecma_string_t *string_p, /**< ecma-string */
 } /* ecma_string_raw_chars */
 
 /**
+ * Checks whether the string equals to the magic string id.
+ *
+ * @return true - if the string equals to the magic string id
+ *         false - otherwise
+ */
+inline bool __attr_always_inline___
+ecma_string_is_magic_id (const ecma_string_t *string_p, /**< property name */
```

Comment:
I think `ecma_compare_ecma_string_to_magic_id` describes better this function. (based on `ecma_compare_ecma_strings`)